settings/src/components/first-time/congratulations.js

```diff
@@ -0,0 +1,95 @@
+/**
+ * WordPress dependencies
+ */
+import { Button } from '@wordpress/components';
+import { useContext } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import { GlobalContext } from '../../script';
+
+/**
+ * Check if the URL is valid. Make sure it stays on wordpress.org.
+ *
+ * @param  url
+ * @return {boolean} Whether it's a valid URL.
+ */
+const isValidUrl = ( url ) => {
+	try {
+		const { hostname } = new URL( url );
+		return hostname.endsWith( 'wordpress.org' );
+	} catch ( exception ) {
+		return false;
+	}
+};
+
+export default function Congratulations() {
+	const {
+		user: { webAuthnEnabled, totpEnabled },
+	} = useContext( GlobalContext );
+
+	const getAuthenticationMethod = () => {
+		if ( webAuthnEnabled && totpEnabled ) {
+			return null;
+		}
+
+		return (
+			<ul>
+				<li>
+					{ totpEnabled && (
+						<a href="https://profiles.wordpress.org/me/profile/edit/group/3/?screen=webauthn">
+							Set up a security key
+						</a>
+					) }
+
+					{ webAuthnEnabled && (
+						<a href="https://profiles.wordpress.org/me/profile/edit/group/3/?screen=totp">
+							Set up one time password
+						</a>
+					) }
+				</li>
+			</ul>
+		);
+	};
+
+	return (
+		<>
+			<p>
+				To ensure the highest level of security for your account, please remember to keep
+				your authentication methods up-to-date, and consult{ ' ' }
+				<a href="//make.wordpress.org/meta/handbook/tutorials-guides/configuring-two-factor-authentication/">
+					our documentation
+				</a>{ ' ' }
+				if you need help or have any questions.
+			</p>
+			<p>
+				We recommend configuring multiple authentication methods to guarantee you always
```

Comment:
"..multiple authentication methods and generating backup codes.." - I know it's redundant but let's emphasize backup codes